packages/fast-development-site-react/src/components/site/index.tsx

```diff
@@ -223,6 +218,16 @@ const styles: ComponentStyles<SiteManagedClasses, DevSiteDesignSystem> = {
     site_paneToc: {
         padding: "0",
         overflow: "auto",
+        "&::-webkit-scrollbar": {
+            width: "6px",
+            height: "6px",
+            overflow: "auto",
+            float: "left",
+        },
+        "&::-webkit-scrollbar-thumb": {
+            background: "#303030",
+            borderRadius: "2px",
+        },
```

Comment:
This seems to be duplicated throughout this package; can we create a utility for this instead of redefining it over and over? Maintenance will be painful if we want to change these.